src-api/openapi-patch.json

```diff
@@ -13,9 +13,7 @@
             },
             "x-examples":{
               "owner-id":{
-                "value":{
-                  "owner-id":"c65b68ef-e73b-4bf2-be9a-7a322a9df150"
-                },
+                "value":"c65b68ef-e73b-4bf2-be9a-7a322a9df150",
```

Comment:
@teesloane this could fix it but I haven't confirmed. Is there a new artifact each build?

Reply:
Yep, if you search for "api/v2/index.html" you should be able to see the output.

Reply:
Thanks 🙏